ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java

```diff
@@ -47,29 +42,27 @@
     private static final int SHADOWMAP_SIZE = 1024;
     private static final int SHADOW_SPLITS = 3;
 
-    private static final Vector3f WORLD_LOCATION = new Vector3f(0, -20, 0);
+    private static final Vector3f WORLD_LOCATION = new Vector3f(300, -20, 0);
 
     private static final Vector3f PLATFORM_LOCATION = new Vector3f(20, -18, -1);
     private static final Vector3f COMMANDER_LOCATION = new Vector3f(23, -14, -1f);
 
     private static final float COMMANDER_ROTATION = -1.5f;
 
-    private static final float PLATFORM_SPEED = 0.2f;
-
-    private static final int LEVEL_PIECES = 5;
-    private static final int DIFFERENT_WORLDS = 5;
-
     private static final float STEERING_ANGLE = (float) (Math.sqrt(2.f) / 2.f);
 
-    private Spatial testPlatform;
-    private LinkedList<Spatial> testWorld;
-    private Spatial testCommander;
+    private Platform platform;
+    private Commander commander;
 
     private DirectionalLight sun;
     private DirectionalLight sun2;
 
     private float steering;
```

Comment:
they are being added to the rootnode, which handles the rendering of these objects, pmd needs to shut it ;)